vapi/simulator/simulator.go

```diff
@@ -224,8 +225,12 @@ func (s *handler) action(r *http.Request) string {
 }
 
 func (s *handler) id(r *http.Request) string {
-	id := path.Base(r.URL.Path)
-	return strings.TrimPrefix(id, "id:")
+	base := path.Base(r.URL.Path)
+	id := strings.TrimPrefix(base, "id:")
+	if id == base {
+		return "" // trigger 404 Not Found w/o id: prefix
```

Comment:
@embano1 with this change, the `id:` prefifx is required.

Reply:
@dougm thx, verified that it works as expected (besides already mentioned #1361 behaviour).